docs/modules/user-guide/partials/proc_tagging-and-pushing-a-registry-container-image.adoc

```diff
@@ -0,0 +1,19 @@
+[id="building-a-custom-devfile-registry_{context}"]
+= Tagging and pushing a registry container image
+
+[role="_abstract"]
+Use the devfile index container image to create a container in which you build, run, and deploy your application.
+
+.Procedure
+
+. Consult the documentation for your container registry and log into the documentation with the Docker CLI.
+. Choose a container registry.
+. Push the devfile index container image to your chosen container registry by running the following commands in order:
+* `$ docker tag -f devfile-index <registry-hostname>/<registry-username>/devfile-index:latest`
```

Comment:
Let's put these commands into a block. Also, note that user-supplied values should be marked up as italic. Putting it all together including the previous line, it should look something like this:

```adoc
. Push the devfile index container image to your chosen container registry:
+
[subs="+quotes,attributes"]
----
$ docker tag -f devfile-index __<registry-hostname>__/__<registry-username>__/devfile-index:latest
$ docker push __<registry-hostname>__/__<registry-username>__/devfile-index:latest
----
```